Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,63 @@
+use num_bigint::{BigUint, ToBigUint};
+use std::io;
+
+// convert input string to BigUint
+fn read_parse(inp: String) -> BigUint {
+    let age: BigUint = inp.trim().parse().expect("Not a positive integer.");
```

Comment:
Misleading error message. If I enter "aa" I would get "Not a positive integer" which is true but misleading.

Reply:
What else should it be then?

Reply:
I think that's the only sentence you need. An age is a positive integer.

Reply:
Looks ok. Not sure what a better error message would be.